### PR TITLE
Bump to Octokit 10.0.0 to fix Int32 overflow issue and add disaster recovery docs

### PR DESCRIPTION
**Pull request by jeffhandley**

### CODE REVIEW

**Review by ericstj — APPROVED** [review]
LGTM - I'd like to dogfood these docs as a user but I don't want to block the PR on that.

**Comment by jeffhandley** [comment]
I followed the DR docs included here to re-deploy all services, and all deployments succeeded. All instances are now working and labeling issues and pulls again.

**Review by mkArtakMSFT — COMMENTED** [review]
Thanks for the writeup, @jeffhandley !

File: Documentation/Implementation.md [hunk 1]
```diff
@@ -33,17 +33,17 @@ This console application consumes the ZIP files produced by `ModelCreator` and u
...
 
-For dotnet/aspnetcore however, this webhook is not active and instead, the aspnetcore repository uses the https://hubbup.io web app to allow for manual area label assignment. Rather than doing automatic assignments, the hubbup app provides a nice UI for the prediction results it receives from [PredictionService](https://github.com/dotnet/issue-labeler/tree/master/src/PredictionService).
+For dotnet/aspnetcore however, this webhook is not active and instead, the aspnetcore repository uses the https://hubbup.io web app to allow for manual area label assignment. Rather than doing automatic assignments, the hubbup app provides a nice UI for the prediction results it receives from [PredictionService](https://github.com/dotnet/issue-labeler/tree/main/src/PredictionService).
```
Comment (danmoseley) [inline Documentation/Implementation.md:42]: It is automatic nowadays?

**Pull request merged**